src/lib/sort/sort.md

```diff
@@ -0,0 +1,31 @@
+The `mdSort` and `md-sort-header` are used, respectively, to add sorting behavior, state, and display
+to tabular data.
```

Comment:
I wouldn't say "behavior", since the components don't actually do any sorting. The directives only capture the state and the display.